assessment_software/src/app/button-component/button.component.html

```diff
@@ -5,7 +5,6 @@
   [style.background-color]="variant === 'Filled' ? background : 'transparent'"
   [style.border]="variant === 'Outlined' ? '1px solid #3056d3' : 'none'"
   [style.color]="color ? color : 'black'"
-  type="button"
```

Comment:
why are you removing type="button"

Reply:
I forgot to undo doing that. I'll put it back